### ISSUE TITLE
The keyboard accessibility icon is not labelled

### ISSUE THREAD

**Issue by alexarnaud** [mate-desktop/mate-desktop#464]
#### Expected behaviour
A tool tip should be displayed to describe the icon when you put the mouse over it. When you move the keyboard focus on the icon with the Orca screen reader enabled, it should announces "NAME OF THE ICON" followed by the type "icon".

#### Actual behaviour
No tool tip is displayed and the Orca screen reader for the blind only says icon because no label is mapped on the icon.

#### Steps to reproduce the behaviour
1) I advise you to try to one in a VM, enabling this icon leads to keyboard weird impact. I had to reset my whole Gsettings parameters.
2) Enable the accessibility icon with this command:
> gsettings set org.mate.accessibility-keyboard enable true
3) Put the mouse over the icon

#### MATE general version
1.20.4 and 1.24.1

#### Package version
1.20.4-2 and 1.24.1-2

#### Linux Distribution
Debian 10 Buster.and Debian Testing
